android_mobile/src/main/java/com/alexstyl/specialdates/donate/DonateActivity.java

```diff
@@ -81,13 +83,13 @@ public void onClick(View v) {
             @Override
             public void run() {
                 scrollToDonate();
-
             }
 
             private void scrollToDonate() {
                 CoordinatorLayout.LayoutParams params = (CoordinatorLayout.LayoutParams) appBarLayout.getLayoutParams();
                 AppBarLayout.Behavior behavior = (AppBarLayout.Behavior) params.getBehavior();
-                behavior.onNestedFling(coordinator, appBarLayout, null, 0, 50, true);
+                if (behavior != null)
```

Comment:
According to this http://stackoverflow.com/a/31182999/1315110 you can set a behavior to the layout. That way you wouldn't have to worry for it being null. Maybe for some versions of Android there is a default behavior set?